MapboxNavigation/PollyVoiceController.swift

```diff
@@ -31,11 +31,21 @@ public class PollyVoiceController: RouteVoiceController {
      */
     public var timeoutIntervalForRequest:TimeInterval = 2
     
+    /**
+     Number of steps ahead of the current step to cache spoken instructions.
+     */
+    public var stepsAheadToCache: Int = 3
```

Comment:
* “Caching” implies reusing things after the first time they’re needed. A more appropriate word for this feature is “prefetching”.
* The name of this property talks about steps instead of spoken instructions – are we fetching individual steps from the Directions API and caching them now?
* By beginning a name with a plural noun, you’re implying a type of `Array`.

How about `countOfStepsForPrefetchingInstructions`?